concurrency-limits-core/src/test/java/com/netflix/concurrency/limits/limiter/AbstractPartitionedLimiterTest.java

```diff
@@ -156,4 +164,54 @@ public void setLimitReservesBusy() {
         Assert.assertEquals(1, limiter.getPartition("batch").getInflight());
         Assert.assertEquals(1, limiter.getInflight());
     }
+
+    @Test
+    public void testBypassPartitionedLimiter() {
+
+        AbstractPartitionedLimiter<String> limiter = (AbstractPartitionedLimiter<String>) TestPartitionedLimiter.newBuilder()
+                .partitionResolver(Function.identity())
+                .partition("batch", 0.1)
+                .partition("live", 0.9)
+                .limit(FixedLimit.of(10))
+                .bypassLimitResolver(new ShouldBypassPredicate())
+                .build();
+
+        for (int i = 0; i < 1; i++) {
+            Assert.assertTrue(limiter.acquire("batch").isPresent());
+            Assert.assertEquals(i+1, limiter.getPartition("batch").getInflight());
+            Assert.assertTrue(limiter.acquire("admin").isPresent());
+        }
+
+        for (int i = 0; i < 9; i++) {
+            Assert.assertTrue(limiter.acquire("live").isPresent());
+            Assert.assertEquals(i+1, limiter.getPartition("live").getInflight());
+            Assert.assertTrue(limiter.acquire("admin").isPresent());
+        }
+    }
```

Comment:
To be very clear about the behavior, let's add
```suggestion
             // Verify that bypassed requests are able to proceed even when the limiter is full
            Assert.assertFalse(limiter.acquire("live").isPresent());
            Assert.assertEquals(10, limiter.getInflight());
            Assert.assertTrue(limiter.acquire("admin").isPresent());
            Assert.assertEquals(10, limiter.getInflight());
    }
```

I understand that we've effectively covered that above, but this will be easier to understand for someone new to the codebase. It also helps defend against a setup issue (eg: if the limit is increased).

Reply:
Added extra check. This behavior is also validated for simple limiters in another test.